client/src/components/results-view/GroupsList.tsx

```diff
@@ -0,0 +1,65 @@
+import { Group, Vote } from "../../types/types";
+import { Paper, Card, Typography, List, ListItem, ListItemText, Divider } from "@mui/material";
+
+interface GroupsListProps {
+    groups: Group[];
+    votes: Vote[];
+}
+
+export const GroupsList: React.FC<GroupsListProps> = ({ groups, votes }) => {
+
+    return (
+        <>
+            <Card component={Paper} sx={{ minWidth: 500 }}>
+                <Typography variant="h4">Wygenerowane grupy</Typography>
+                <List sx={{ display: 'flex', flexDirection: 'row', alignItems: 'flex-start' }}>
+                    {groups.map(group => {
```

Comment:
Po kliknięciu wyrzuca
![eror_map](https://github.com/kBlazewicz/group-enroll/assets/74615934/f102e263-0fff-48b2-94e8-917ed4f0b582)
Może dlatego, że formularz studenta nie pozwala póki co wprowadzić więcej niż jednego studenta, ale i tak pasowałoby to jakoś zablokować. (np. alert jeśli studentów jest za mało)